### ISSUE TITLE
parler triggers RemovedInDjango18Warning

### ISSUE THREAD

**Issue by FinalAngel** [edoburu/django-parler#94]
```
.../parler/forms.py:163: RemovedInDjango18Warning: Creating a ModelForm without either the 'fields' attribute or the 'exclude' attribute is deprecated - form EventAdminForm needs updating
  return super(TranslatableModelFormMetaclass, mcs).__new__(mcs, name, bases, attrs)
```

**Comment by vdboor** [comment]
Closing, parler supports 1.8 / 1.9 for a while now.

**Comment by FinalAngel** [comment]
@vdboor has the warning been resolved? This was not actually an error but rather a warning generated in the console.

**Comment by vdboor** [comment]
I'm not getting this warning anymore. It's fixed with the Django 1.8 support fixes (as `RemovedInDjango18Warning` became an error in 1.8)

**Comment by FinalAngel** [comment]
:+1: awesome, thanks for the feedback @vdboor
